### PR TITLE
color changes

### LINKED ISSUE #143: Fix striped tables and panel footers
For screenshots see #139.

### PR DESCRIPTION
**Pull request by janno42**
fix #143; change colors of striped tables, panel footers and primary buttons

### CODE REVIEW

File: _1327/static/less/1327.less [hunk 1]
```diff
@@ -220,3 +220,11 @@ footer a {
 table.table-narrow {
 	max-width: 400px;
 }
+
+// Overwrite bootstrap table coloring
+.table-striped > tbody > tr:nth-of-type(odd) {
+	background-color: @white;
+}
```
Comment (invliD) [inline _1327/static/less/1327.less]: Please modify ``@table-bg-accent`` and remove this part.

File: _1327/static/less/1327.less [hunk 2]
```diff
@@ -220,3 +220,11 @@ footer a {
...
+.table-striped > tbody > tr:nth-of-type(even) {
+	background-color: @gray-lighter;
+}
```
Comment (invliD) [inline _1327/static/less/1327.less]: Please introduce a new variable for this color (right next to ``@table-bg-accent``), put in a comment that it's not a bootstrap variable, and move this block to the bootstrap fixes at the top of the file.

File: _1327/static/less/1327.less (hunk 1 above)
Comment (janno42) [inline _1327/static/less/1327.less]: i still need the definition for even rows. i thought having both at the same place would be nice. what do you think?

Comment (invliD) [inline _1327/static/less/1327.less]: See my other comment below.

**Comment by janno42** [comment]
updated

**Comment by invliD** [comment]
:+1:

**Pull request merged**